apps/team-awesome-web/src/components/team-members/TeamMembers.tsx

```diff
@@ -27,7 +27,14 @@ export const TeamMembers = ({ ...props }: TeamMembersProps) => {
   return (
     <Styled.TeamMembers>
       <Styled.Actions>
-        <Select sortList="teamMembers" direction />
+        <Select
+          sortList="teamMembers"
+          sortArray={teamMembers}
+          setSortedResults={
+            setSortedTeamMembers as Dispatch<SetStateAction<Types.TeamMember[] | Types.TimeEntry[]>>
```

Comment:
Dit is een vrij ongebruikelijke plek om je state setter te typen. Is het nodig?

Reply:
I completely agree, I was given a fix by a teacher to type these more accurately at their declaration. Will process that here as well